src/main/webapp/i18n/en/exercise.json

```
@@ -241,6 +241,14 @@
                     "customMin": "The number of bonus points must be at least 0!",
                     "customMax": "The number of bonus points may not exceed 9999!"
                 },
+                "minTeamSize": {
+                    "min": "The lower bound for the minimal team size is 1 person.",
+                    "max": "The upper bound for the minimal team size are 99 people."
+                },
+                "maxTeamSize": {
+                    "min": "The lower bound for the maximal team size is 1 person.",
+                    "max": "The upper bound for the maximal team size are 99 people."
```

(here also the english suggestion)
```suggestion
                    "min": "The minimal team size must not be less than 1 person.",
                    "max": "The minimal team size must not exceed 99 people."
                },
                "maxTeamSize": {
                    "min": "The maximal team size must not be less than 1 person.",
                    "max": "The maximal team size must not exceed 99 people."
```